modules/platforms/cpp/ignite/odbc/app/application_data_buffer.cpp

```diff
@@ -892,22 +841,27 @@ conversion_result application_data_buffer::put_time(const ignite_time& value)
         case odbc_native_type::AI_TTIMESTAMP:
         {
             auto* buffer = reinterpret_cast<SQL_TIMESTAMP_STRUCT*>(data_ptr);
+            memset(buffer, 0, sizeof(*buffer));
```

Comment:
Is this cleanup necessary? We seemingly override the entire buffer below.

Reply:
Such a cleanup considered to be a good practice in C world, because you can not be sure that for example on other platform or in the next version of standard they will not add some new fields.